src/fft_common.f90

```diff
@@ -75,93 +88,142 @@ subroutine fft_init_arg(pencil)     ! allow to handle Z-pencil input
 end subroutine fft_init_arg
 
 ! Initialise the FFT library to perform arbitrary size transforms
-subroutine fft_init_general(pencil, nx, ny, nz)
+subroutine fft_init_general(pencil, nx, ny, nz, opt_inplace, opt_inplace_r2c, opt_inplace_c2r)
```

Comment:
Would `init_one` or `init_single` now be a better name?

Reply:
Fixed in https://github.com/2decomp-fft/2decomp-fft/pull/302/commits/7033c26a9c4b2da305a0d16b34088a931f8bb1b7